test/worker.spec.js

```diff
@@ -15,7 +15,7 @@ module.exports.addTests = function({testRunner, expect}) {
       expect(await worker.evaluate(() => self.workerFunction())).toBe('worker function result');
 
       await page.goto(server.EMPTY_PAGE);
-      expect(page.workers()).toEqual([]);
+      expect(page.workers().length).toBe(0);
```

Comment:
I don't get it. Whats the change?

Reply:
when this test fails, the output is cryptic since Worker object is not nicely serializable.

Reply:
Fair